addons/stock/wizard/stock_picking_return.py

```diff
@@ -95,9 +95,10 @@ def _create_returns(self):
             'move_lines': [],
             'picking_type_id': picking_type_id,
             'state': 'draft',
-            'origin': self.picking_id.name,
+            'origin': "Return of " + self.picking_id.name,
```

Comment:
This needs to be translatable.

Reply:
When we return any products from delivery order, it needs to be set like 'Return of' particular delivery order.

Reply:
Sure but it still needs to be translatable, "Return of X" is sensible if your Odoo is in english, but it makes no sense if your Odoo is configured in Korean, or Gujarati, or Mexican Spanish, in those cases you want the equivalent phrase in the relevant grammar and script.